applications/source/cdc-debezium-source/src/test/java/org/springframework/cloud/stream/app/source/cdc/CdcSourceDatabasesIntegrationTest.java

```diff
@@ -57,13 +62,12 @@ public class CdcSourceDatabasesIntegrationTest {
 					"cdc.schema=false",
 					"cdc.flattening.enabled=true",
 					"cdc.stream.header.offset=true",
-					// "cdc.config.database.server.id=85744",
 					"cdc.config.database.server.name=my-app-connector",
 					"cdc.config.database.history=io.debezium.relational.history.MemoryDatabaseHistory");
 
 	@Test
 	public void mysql() {
-		GenericContainer debeziumMySQL = new GenericContainer<>("debezium/example-mysql:1.7.1.Final")
+		GenericContainer debeziumMySQL = new GenericContainer<>("debezium/example-mysql:1.9.6.Final")
```

Comment:
Wdyt about moving the docker image name up to a constant that all of the tests can reference. It is currently in many places. I believe it can be either a `DockerImageName` or a `String` ?